NuGet.Config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <clear />
```

Comment:
Curious why this is needed. Usually it's there to prevent the machine-wide NuGet.Config from having an effect

Reply:
It should not be there, it's just for testing.